awsconfigs/common/istio-ingress/overlays/cognito/ingress.yaml

```diff
@@ -7,3 +7,4 @@ metadata:
     alb.ingress.kubernetes.io/auth-idp-cognito: '{"UserPoolArn":"$(CognitoUserPoolArn)","UserPoolClientId":"$(CognitoAppClientId)", "UserPoolDomain":"$(CognitoUserPoolDomain)"}'
     alb.ingress.kubernetes.io/certificate-arn: $(certArn)
     alb.ingress.kubernetes.io/listen-ports: '[{"HTTPS":443}]'
+    alb.ingress.kubernetes.io/target-type: ip
```

Comment:
Why not add it to base ingress manifest?

https://github.com/awslabs/kubeflow-manifests/blob/main/awsconfigs/common/istio-ingress/base/ingress.yaml

Reply:
moved the annotation into base instead.